config/services.yaml

```diff
@@ -31,15 +39,10 @@ services:
 
   Neusta\Pimcore\FixtureBundle\Locator\:
     resource: '../src/Locator/*'
-
-  Neusta\Pimcore\FixtureBundle\FixtureLoader\FixtureLoader:
-    arguments:
-      $fixtureLocator: '@Neusta\Pimcore\FixtureBundle\Locator\AllFixturesLocator'
-
-  Neusta\Pimcore\FixtureBundle\FixtureLoader\SelectiveFixtureLoader:
-    arguments:
-      $fixtureLocator: '@Neusta\Pimcore\FixtureBundle\Locator\NamedFixtureLocator'
     public: true
 
+  Neusta\Pimcore\FixtureBundle\Profiler\:
+    resource: '../src/Profiler/*'
```

Comment:
I think this should only be loaded in `debug` mode, because the profiler is a development tool.